### PR TITLE
USWDS-Site: Documentation updates for 3.8.1

### LINKED ISSUE #2652: USWDS-Site: Add documentation updates for 3.8.1
## Summary
- Update the 3.8.0 references in `_config.yml` to 3.8.1.
- Add 3.8.1 changelogs
- Update uswds to version 3.8.1

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary

- Updated config file uswds version and package hash url
- Added 3.8.1 changelogs 
- Updated uswds version in package.json

## Related issue

Closes #2652 

## Preview link

[Homepage →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.1/)

- [Button group changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.1/components/button-group/#latest-updates)
  - uswds/uswds#5885
  - Changelog: #2644 
- [Documentation page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.1/documentation/developers/)
  - uswds PR: uswds/uswds#5871
  - uswds PR: uswds/uswds#5915
- [Footer changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.1/components/footer/#latest-updates)
    - uswds PR: uswds/uswds#5930
    - Changelog: #2676
- [Identifier changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.1/components/identifier/#latest-updates)
  - uswds PR: uswds/uswds#5857
  - Changelog: #2610 
- [Input mask changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.1/components/input-mask/#latest-updates)
  - uswds PR: uswds/uswds#5518
  - Changelog: #2658
- [Tooltip changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.1/components/tooltip/#latest-updates)
  - uswds PR: uswds/uswds#5909
  - Changelog: #2660
  - uswds PR: uswds/uswds#5919
  - Changelog: #2661
- [Validation changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.1/components/validation/#latest-updates)
  - uswds PR: uswds/uswds#5835
  - Changelog: #2555

## No documentation changes needed

- USWDS - Dependencies: POAM April '24
  - uswds PR: uswds/uswds#5854
  - Changelog: N/A - Not a user facing change
- USWDS - Dependencies: Update eslint, prettier, stylelint and dependencies
  - uswds PR: uswds/uswds#5782
  - Changelog: N/A - Not a user facing change
- USWDS - Storybook: Create form error story
  - uswds PR: uswds/uswds#5879
  - Changelog: N/A - Not a user facing change
- USWDS - Utilities: Update inaccurate comments
  - uswds PR: uswds/uswds#5859
  - Changelog: N/A - SCSS comments; not user facing
- USWDS - In-page navigation: Update reference to data-heading-elements sttribute
  - uswds PR: uswds/uswds#5856
  - Changelog: N/A - Not a user facing change

### Confirm update to 3.8.1
- Confirm that the "Download" button in the header shows the correct version number
- On the download page, confirm:
    - The version number is correct
    - The listed SHA-256 hash matches the release hash
- Confirm that the package.json uses uswds version 3.8.1

### Confirm changelogs
- Confirm all user-facing changes have a related changelog entry
- Confirm that changelog entries are accurate and make sense
- Confirm the changelog dates and version numbers are accurate

### Confirm component updates
- Confirm that the Spanish versions of the identifier component do not include the word "An"

### CODE REVIEW

File: _data/changelogs/component-tooltip.yml [hunk 1]
```diff
@@ -5,6 +5,21 @@ type: component
 changelogURL:
 
 items:
+  - date: 2024-05-31
+    summary: Updated component behavior to close active tooltips when the `escape` key is pressed.
+    summaryAdditional: This allows the component to meet the "dismissible" standard outlined in [WCAG 1.4.13](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus).
+    affectsJavascript: true
+    githubPr: 5909
+    githubRepo: uswds-site
```
Comment (amyleadem) [inline _data/changelogs/component-tooltip.yml:13]:
```suggestion
    githubRepo: uswds
```

File: _data/changelogs/component-identifier.yml [hunk 2]
```diff
@@ -2,6 +2,14 @@ title: Identifier
 type: component
 changelogURL:
 items:
+  - date: 2024-05-31
+    summary: Fixed a bug that mistakenly added the word "An" to Spanish variants in the component preview.
```
Comment (amyleadem) [inline _data/changelogs/component-identifier.yml:6]: Addition for clarity
```suggestion
    summary: Fixed a bug that mistakenly added the English word "An" to Spanish variants in the component preview.
```

File: _data/changelogs/component-tooltip.yml [hunk 3]
```diff
@@ -5,6 +5,21 @@ type: component
 changelogURL:
 
 items:
+  - date: 2024-05-31
+    summary: Updated component behavior to close active tooltips when the `escape` key is pressed.
+    summaryAdditional: This allows the component to meet the "dismissible" standard outlined in [WCAG 1.4.13](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus).
+    affectsJavascript: true
```
Comment (amyleadem) [inline _data/changelogs/component-tooltip.yml:11]:
```suggestion
    affectsJavascript: true
    affectsAccessibility: true
```

File: _data/changelogs/component-tooltip.yml [hunk 4]
```diff
@@ -5,6 +5,21 @@ type: component
...
+    githubPr: 5909
+    githubRepo: uswds
+    versionUswds: 3.8.1
+  - date: 2024-05-31
+    summary: Updated component behavior to allow users to hover over tooltip content.
+    summaryAdditional: This allows the component to meet the "hoverable" standard outlined in [WCAG 1.4.13](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus).
+    affectsJavascript: true
```
Comment (amyleadem) [inline _data/changelogs/component-tooltip.yml:18]:
```suggestion
    affectsAccessibility: true
    affectsJavascript: true
```

File: _data/changelogs/component-identifier.yml [hunk 5]
```diff
@@ -2,6 +2,14 @@ title: Identifier
...
+    summaryAdditional:
+    affectsGuidance: true
+    affectsContent: true
```
Comment (amyleadem) [inline _data/changelogs/component-identifier.yml:9]:
```suggestion
    affectsContent: true
    affectsMarkup: true
```

Comment (thisisdano) [reply _data/changelogs/component-identifier.yml:9]: Lets reserve the markup key only for changes that affect the structure of the component

**Review by thisisdano — APPROVED** [review]

**Pull request merged**